### PR TITLE
Bump parallel catchup memory to 6GB

### PR DESCRIPTION
**Pull request by marta-lokhova**

### CODE REVIEW

**Review by sisuresh — APPROVED** [review]
The cluster should be able to handle this, so we might as well merge this in for now while we look into the memory usage.

File: src/FSLibrary/StellarKubeSpecs.fs [hunk 1]
```diff
@@ -148,7 +148,7 @@ let SimulatePubnetTier1PerfCoreResourceRequirements : V1ResourceRequirements =
 let ParallelCatchupCoreResourceRequirements : V1ResourceRequirements =
     // When doing parallel catchup, we give each container
     // 1200MB RAM, 0.25 vCPUs, and 35 GB of disk bursting to 2vCPU, 4500MB and 40 GB
```
Comment (SirTyson) [inline src/FSLibrary/StellarKubeSpecs.fs:150]: Nit: "4500MB" should be "6000MB"

**Review by SirTyson — APPROVED** [review]
LGTM

File: src/FSLibrary/StellarKubeSpecs.fs (hunk 1 above)
Comment (marta-lokhova) [reply src/FSLibrary/StellarKubeSpecs.fs]: Fixed.

**Pull request merged**